packages/components/src/tooltip/index.tsx

```diff
@@ -128,7 +128,12 @@ function UnforwardedTooltip(
 			{ isOnlyChild && ( text || shortcut ) && (
 				<Ariakit.Tooltip
 					{ ...restProps }
-					className={ clsx( 'components-tooltip', className ) }
+					className={ clsx(
+						'components-tooltip',
+						{ 'move-up': text?.toLowerCase().includes( 'up' ) },
+						{ 'move-down': text?.toLowerCase().includes( 'down' ) },
```

Comment:
Inferring tooltip behavior or custom visual differences based on specific tooltip text doesn't make much sense to me. 

The problem is that `Tooltip` is a generic tooltip component, while `Move up" and "Move down" are coming from specific tooltip usages inside the block editor. 

With that in mind, if we're to add custom styles based on that, it shouldn't be in the generic component but should rather be in the consumer component.